src/Uno.UI/UI/Xaml/Controls/DatePicker/DatePickerFlyoutPopupPanel.iOS.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Windows.Foundation;
+using Windows.UI.Xaml.Controls.Primitives;
+
+namespace Windows.UI.Xaml.Controls
+{
+	internal partial class DatePickerFlyoutPopupPanel : FlyoutBasePopupPanel
+	{
+		private readonly FlyoutBase _flyout;
+
+		public DatePickerFlyoutPopupPanel(FlyoutBase flyout) : base(flyout)
+		{
+			_flyout = flyout;
+		}
+
+		protected override Rect CalculateFlyoutPlacement(Size desiredSize)
+		{
+			// place the content at full width in the bottom of the screen 
```

Comment:
Make a note that this is expected for iOS, bonus points if you can link to an official source.

Reply:
done